fxa-auth-db-server/index.js

```diff
@@ -89,6 +89,7 @@ function createServer(db) {
       return db.deleteEmail(req.params.id, req.params.email)
     })
   )
+  api.get('/account/emails/:id', withIdAndBody(db.getEmail))
```

Comment:
I change that, wanted to reuse `withIdAndBody` helper which requires the param to be named `id`, but it is not really needed.